Chore: Release notes 2021.01 (#5381)
patternfly/patternfly-react: packages/react-docs/RELEASE-NOTES.md

- **Checkbox:** 
  - Updated to show error message if id is missing ([#5292](https://github.com/patternfly/patternfly-react/pull/5292))
- **Chip group:**
  - Added event handler prop for overflow-chip click ([#5272](https://github.com/patternfly/patternfly-react/pull/5272))
- **Code editor:** 
  - Added new code editor component ([#5294](https://github.com/patternfly/patternfly-react/pull/5294))
- **Context selector:** 
  - Removed isHovered from item unit tests ([#5306](https://github.com/patternfly/patternfly-react/pull/5306))
- **Drawer:** 
  - Added resize callback ([#5319](https://github.com/patternfly/patternfly-react/pull/5319))
  - Fixed dragging problem on iframe when resizing drawer ([#5331](https://github.com/patternfly/patternfly-react/pull/5331))
  - Improved cursor responsiveness ([#5328](https://github.com/patternfly/patternfly-react/pull/5328))
  - Fixed initial width not respecting min/max widths ([#5313](https://github.com/patternfly/patternfly-react/pull/5313))
  - Fixed to stop panel content unmounting before collapse ([#5330](https://github.com/patternfly/patternfly-react/pull/5330))

> - **DualListSelector:** 

Reviewer: ```suggestion
- **Dual list selector:** 
```

  - Added type=button to DualListSelectorListItem button ([#5266](https://github.com/patternfly/patternfly-react/pull/5266))
  - Fixed to not pass custom handlers to root div ([#5308](https://github.com/patternfly/patternfly-react/pull/5308))
- **Empty state** 
  - Cleaned up errors in tests ([#5302](https://github.com/patternfly/patternfly-react/pull/5302))
- **Jumplinks:** 
  - Updated to allow hrefs with spaces ([#5223](https://github.com/patternfly/patternfly-react/pull/5223))
  - **Menu:** 

>     - Fixed sentence case for menu example ([#5262](https://github.com/patternfly/patternfly-react/pull/5262))

Reviewer 2: Remove tab on these lines

- **Popover:** 
  - Updated to hide Popover when its trigger is scrolled out of view ([#5318](https://github.com/patternfly/patternfly-react/pull/5318))
- **Radio:** 
  - Updated to show error message if id is missing ([#5292](https://github.com/patternfly/patternfly-react/pull/5292))
- **Search input:** 
  - Updated to hide clear button when onClear function is not set ([#5230](https://github.com/patternfly/patternfly-react/pull/5230))
- **Select:** 
  - Fixed crash when opened using enter with disabled options ([#5258](https://github.com/patternfly/patternfly-react/pull/5258))
  - Cleaned up errors in tests ([#5302](https://github.com/patternfly/patternfly-react/pull/5302))
- **Simple list:** 
  - Added uncontrolled Simple list support ([#5134](https://github.com/patternfly/patternfly-react/pull/5134))
- **Slider:** 
  - Added new slider component ([#5358](https://github.com/patternfly/patternfly-react/pull/5358))
- **Toggle group:** 
  - Fixed bug with ToggleGroupItem button being clicked in form in modal closing modal ([#5259](https://github.com/patternfly/patternfly-react/pull/5259))
- **Toolbar:** 

>   - Added log viewer toolbar example ([#5196](https://github.com/patternfly/patternfly-react/pull/5196)

Reviewer: ```suggestion
  - Added log viewer toolbar example ([#5196](https://github.com/patternfly/patternfly-react/pull/5196))
```

- Fixed chart legend tooltip shown with no data ([#5296](https://github.com/patternfly/patternfly-react/pull/5296))
- Fixed incorrect colorScale prop type ([#5303](https://github.com/patternfly/patternfly-react/pull/5303))
- Updated so theme font must use token value ([#5301](https://github.com/patternfly/patternfly-react/pull/5301))
-  Fixed tooltip pointerLength error ([#5245](https://github.com/patternfly/patternfly-react/pull/5245))

### Table
  - Updated to make empty state consistent across demos and examples ([#5298](https://github.com/patternfly/patternfly-react/pull/5298))
  - Fixed scope axe issue ([#5253](https://github.com/patternfly/patternfly-react/pull/5253))

### Topology
  - Adjusted topology node layout when new nodes are added ([#5249](https://github.com/patternfly/patternfly-react/pull/5249))
 - Prevented context sub menus from overflowing viewport ([#5271](https://github.com/patternfly/patternfly-react/pull/5271))

### Other
- **Chore:** 
  - Added linting rule to disallow nested ternaries ([#5255](https://github.com/patternfly/patternfly-react/pull/5255))
- **Demos:** 

>   - Enable dstatic drawer open on mobile view in primary detail demo ([#5311](https://github.com/patternfly/patternfly-react/pull/5311))

Reviewer: ```suggestion
  - Enabled static drawer open on mobile view in primary detail demo ([#5311](https://github.com/patternfly/patternfly-react/pull/5311))
```
